package.json

```diff
@@ -67,8 +72,42 @@
   },
   "config": {
     "commit-message-validator": {
-      "pattern": "^(#([0-9]+).*)",
-      "errorMessage": "Your commit message must start with issue number e.g. #39"
+      "pattern": "^(fix: #|feat: #|perf: #)[0-9].*",
+      "errorMessage": "Your commit should has this format> fix or feat or perf: #45 message"
```

Comment:
Your commit message needs to start with fix:, feat:, or perf: followed by issue number, e.g. fix: #43 any commit message